### PR TITLE
Run Python 2.6 on trusty and add Python 3.8

### PR DESCRIPTION
**Pull request by cclauss**
@baztian Is https://virtualenv.pypa.io/en/latest/changelog.html#v20-0-11-2020-03-18 the remaining problem?

### CODE REVIEW

**Comment by baztian** [comment]
@cclauss  as far as I remember https://github.com/jythontools/jython/issues/160 was (also?) an issue.

**Comment by baztian** [comment]
Also to make deploy to PyPI work we need @sunng87 to verify his email. #54

File: .travis.yml [hunk 1]
```diff
@@ -25,7 +28,7 @@ script:
 - python test/testsuite.py
 deploy:
   provider: pypi
-  user: sunng
+  username: sunng
```
Comment (baztian) [inline .travis.yml:31]: Why did you change user to username? This is not the way it's described in https://docs.travis-ci.com/user/deployment/pypi/

**Comment by cclauss** [comment]
Screenshot 2020-05-25 at 15 44 00

**Comment by cclauss** [comment]
Thx... travis-ci/docs-travis-ci-com#2803

File: .travis.yml (hunk 1 above)
Comment (cclauss) [reply .travis.yml:31]: Thx... travis-ci/docs-travis-ci-com#2803

**Pull request merged**